### ISSUE TITLE
DNS failed to resolve after system restart

### ISSUE THREAD

**Issue by PsPmax** [quidsup/notrack#187]
Hey Quids!
It seems I have run into a snag with NoTrack recently and maybe found an issue? This could be my wrong doing considering i'm not THAT experienced with linux and computer networking. But here's what's going on. I was running NoTrack on my OrangePi PC for about two weeks without any problems. It was working great with the Lubuntu 14.04 image OrangePi supplied. A couple nights ago I had to restart the system using the built in option within the NoTrack web interface. I've done this before so I didn't think anything of it. When the OrangePi restarted, any systems using it as a DNS server couldn't resolve URLs. To be honest, I thought something got screwed up and decided to just format the OrangePi and start from scratch. I installed the same OS on the Pi, set a static IP, installed NoTrack, and pointed all my devices at the Pi to use it as a DNS server. It worked great until I restarted the Pi like 10 minutes later and did the same thing the previous night. What's weird is the Pi's internet connection was working perfectly. What’s also weird is I can pause NoTrack blocking for X amount of minutes and all the other devices can access the web again. At this point I've tried a couple different operating systems and they're all yielding the same result. I can install NoTrack just fine and have it work blocking sites on the other devices until it's time for the Pi to reboot and all hell breaks loose. I'm sorry if this is the wrong place to post this, I’m not sure if you have a forum or something where this message would be more appropriate. I've just been scratching my head trying to figure this out, and any help would be greatly appreciated! I loved NoTrack when I didn't screw it up and really want it back.

Thank you so much for your time everyone!

**Comment by quidsup** [comment]
I don't have a forum, so a bug post is fine.
Sounds like the issue lies with Dnsmasq, and it sounds very familiar to an issue that has been brought to my attention #65 
Although the issue with Dnsmasq started in 16.04, which would post date your system.

**Comment by PsPmax** [comment]
That's what I was thinking to be honest. I'll try the proposed solution in that post! If that doesn't work I'll try an OS that has a server version for the OrangePi. Thanks for your guidance Quids!

**Comment by PsPmax** [comment]
This is probably a long overdue response to my original post. I've been super busy with work and school, and the Orange Pi was placed on the back burner. That being said, I finally got around to fixing the issue I was having. I tried the solution in issue #65 which worked with some success. What really ended up fixing the original problem was just pausing blocking when the system first booted, then resuming blocking.  Doing that made everything work correctly. Just to double check though, I ran windows command prompt and started pinging websites on the block list. Sure enough NoTrack was blocking them. Very weird I know, to be honest I'm just happy NoTrack is back. So if anyone else has this very strange issue, try solution #65 first, then just try pausing and then unpausing blocking. Quids, I can try and replicate this issue for you, and send you any necessary files if that would help?
